src/components/payment/payment-mixin.ts

```diff
@@ -60,8 +60,7 @@ export default class PaymentMixin extends Vue {
   }
 
   get paymentSummaries () {
-    const payments = this.$store.getters[paymentTypes.GET_PAYMENTS]
-    const summaries = payments.map(payment => {
+    const summaries = this.payments?.map(payment => {
```

Comment:
When the app started, I was getting an error about `this.payments` being undefined. I don't recall this before so I'm looking to see if I broke anything. For now, the optional chaining resolves it.